### ISSUE TITLE
PLY format and alpha channel

### ISSUE THREAD

**Issue by ua4192** [cnr-isti-vclab/nexus#67]
Hi.

First to say sorry if this is not the appropiate channel for raising questions. If it is not, please forward me the appropiate one and I will close this ticket.

My question is directly linked with PLY files that are turned into NXS files and after to NXS using nxsbuild and nxsedit commands. I am using PLY in its binary format.

As I have read, the PLY format has no specific property for alpha  channel (only red, green and blue properties). I can change my PLY Exporter in order to create a new property and add alpha channel but my question is ¿Does your framework support ply files with RGBA or not?

In fact, from the PLYExporter and PLYLoader from THREEJS there is not currently support for alpha channel.

Looking into your src files at plyloader.cpp I have seen that you are using an extra vertex property named alpha.

I guess that simply exporting my ply files with this new property would be enough?

The modifications under PLYExporter are quite simple to enable this transparency support.

Again many thanks for your support.

Best regards

**Comment by ponchio** [comment]
Yes alpha channel in color per vertex is supported, even in compression.
It has been used sometimes for ambient occlusion.

If something doesn't work properly, contact me.

**Comment by ua4192** [comment]
Many thanks.

I will keep you informed.

Best regards
